src/Workspaces/Core/Portable/Workspace/Solution/ProjectDependencyGraph_RemoveProject.cs

```diff
@@ -73,12 +77,15 @@ private static ImmutableDictionary<ProjectId, ImmutableHashSet<ProjectId>> Compu
             // 'projectId' from the set of references if it exists.
             foreach (var (id, _) in existingForwardReferencesMap)
             {
-                builder.MultiRemove(id, removedProjectId);
+                foreach (var removedProjectId in removedProjectIds)
+                    builder.MultiRemove(id, removedProjectId);
             }
         }
 
         // Finally, remove 'projectId' itself.
```

Comment:
This comment wasn't updated in a prior refactoring, but if you're renaming it again maybe we can try getting it right.